pkg/network/ebpf/c/protocols/protocol-classification-defs.h

```diff
@@ -25,6 +42,7 @@ typedef enum {
     PROTOCOL_HTTP,
     PROTOCOL_HTTP2,
     PROTOCOL_TLS,
+    PROTOCOL_AMQP,
```

Comment:
that's wrong
you should open a PR in the [agent-payload repo](https://github.com/DataDog/agent-payload/blob/master/proto/process/agent.proto) to add the protocol to the list.
the value here should match the value in the protobuf.

Reply:
done: https://github.com/DataDog/agent-payload/pull/210

Reply:
still, the value here will be `5` while it should be `8`

```suggestion
    PROTOCOL_AMQP = 8,
```